### DISCUSSION TITLE
Metrics on fargate clusters do not sum up to the correct amounts on the final console output

### DISCUSSION THREAD

**Discussion by ethan-coles** [artilleryio/artillery#3329] [Artillery]
I am having an issue where let's say I run a test with an `arrivalCount: 200` on 5 AWS clusters totaling 1000 virtual users. The summary section at the end of console output would display a number less than 1000 for the metric `vusers.created`, yet when viewing each of the 5 CloudWatch logs, they all show the correct amount of 200 for each. Other metrics also display this behaviour.

The reasoning for opening a discussion is I am unable to reproduce in a simple scenario. A current workaround so far has been scaling out horizontally which leads me to assume it's correlated with high client CPU/memory usage.

**Comment by hassy** [comment]
thanks @ethan-coles! we've had a couple of reports from different users of this issue, we're looking into it! It's very difficult to reproduce reliably as you noted :)
